### PR TITLE
Bug-1178040 Edit Profile and Preferences Tabs added

### PR DESCRIPTION
**Pull request by SebastinSanty**
screen shot 2016-07-07 at 11 28 40 am

Added the "Edit Profile and Preferences" in My Profile as a button and renamed in Header Menu. Also removed the (change) after name to avoid confusion.

### CODE REVIEW

**Comment by SebastinSanty** [comment]
Description: See [Bug 1178040](https://bugzilla.mozilla.org/show_bug.cgi?id=1178040)

File: template/en/default/global/header.html.tmpl [hunk 1]
```diff
@@ -315,7 +315,7 @@
                 <li><a href="user_profile">My Profile</a></li>
                 <li><a href="page.cgi?id=user_activity.html&amp;action=run&amp;who=[% user.login FILTER uri %]">My Activity</a></li>
                 <li><a href="request.cgi?requester=[% user.login FILTER uri %]&amp;requestee=[% user.login FILTER uri %]&amp;do_union=1&amp;group=type&amp;action=queue">My Requests</a></li>
-                <li><a href="userprefs.cgi">Preferences</a></li>
+                <li><a href="userprefs.cgi">Edit Profile & Preferences</a></li>
```
Comment (dylanwh) [inline template/en/default/global/header.html.tmpl]: I think this makes the menu too wide. Probably leave it as "Preferences".
The edit button makes it discoverable enough.

File: extensions/UserProfile/template/en/default/pages/user_profile.html.tmpl [hunk 2]
```diff
@@ -73,12 +70,16 @@
       [% " (disabled)" UNLESS target.is_enabled %]
     </td>
   </tr>
+  <tr>
+    <th>&nbsp;</th>
+    <td colspan="2">
+      [% IF target.id == user.id %]
+      <a href="userprefs.cgi?tab=account"><input type = "button" value = "Edit Profile & Preferences"></a>
```
Comment (dylanwh) [inline extensions/UserProfile/template/en/default/pages/user_profile.html.tmpl]: nit: remove the spaces between = for html attributes: type="button"
I think the button can just say "Edit Profile". GitHub itself does this same thing --
it has a profile and a settings page, and on the profile there is an Edit Profile button that takes you to the Profile section of settings.

**Comment by SebastinSanty** [comment]
Updated with changes.

**Comment by SebastinSanty** [comment]
screen shot 2016-07-07 at 8 45 12 pm

**Comment by SebastinSanty** [comment]
screen shot 2016-07-09 at 2 52 58 am
Just for preview.

**Comment by dylanwh** [comment]
This patch was commited with a minor change: https://github.com/mozilla-bteam/bmo/commit/a446cca4e59bb1708e4556a7e5b52dc6a47dc9eb#diff-bf66d9159dca342b168735f3e4ed3e35R49

**Pull request closed without merging**